contracts/contracts/strategies/BaseConvexMetaStrategy.sol

```diff
@@ -48,7 +48,7 @@ abstract contract BaseConvexMetaStrategy is BaseCurveStrategy {
     uint128 crvCoinIndex;
     uint128 mainCoinIndex;
 
-    int256[30] private __reserved;
```

Comment:
Why is this changed to 41?

Reply:
As per audit recommendation that base contracts take up 50 storage slots combining local variables with the __reserve gap.